### PR TITLE
Change region to eu west 1 + add CI

### PR DESCRIPTION
**Pull request by OzTK**

### CODE REVIEW

File: ci/release.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "TODO: Push to ECR"
+
+echo "TODO: Push to ECS"
```
Comment (jarimatti-q) [inline ci/release.sh]: Note for future: pushing to ECS could be done as CodePipeline.

Comment (OzTK) [reply ci/release.sh]: Oh nice, I'll do it this way then!

**Comment by OzTK** [comment]
Sorry, forgot the label but this is still WIP

**Comment by OzTK** [comment]
@jarimatti-q could you run `deploy_ci.sh` to verify that the stack deploys successfully? That should then add the checks in the PR

**Comment by jarimatti-q** [comment]
Will do.

**Comment by jarimatti-q** [comment]
Stack deploy failed, logs here: https://eu-west-1.console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/stackinfo?filteringText=&filteringStatus=active&viewNested=true&hideStacks=false&stackId=arn%3Aaws%3Acloudformation%3Aeu-west-1%3A973202160551%3Astack%2Fbuild-aws-codepipeline-dashboard%2Fb3feb000-437c-11ea-b3f2-0af8eed4b1ec

**Comment by OzTK** [comment]
> Stack deploy failed, logs here: https://eu-west-1.console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/stackinfo?filteringText=&filteringStatus=active&viewNested=true&hideStacks=false&stackId=arn%3Aaws%3Acloudformation%3Aeu-west-1%3A973202160551%3Astack%2Fbuild-aws-codepipeline-dashboard%2Fb3feb000-437c-11ea-b3f2-0af8eed4b1ec

Simple typo in the repo name 🤦‍♂

**Comment by jarimatti-q** [comment]
The stack is now there. The build there is not triggered yet but this seems mergeable.

**Comment by OzTK** [comment]
@jarimatti-q: Needed to make one more change. Stack needs to be re-depoyed. I think there's some config to change in the repo to have the checks appear here

**Comment by jarimatti-q** [comment]
Redeploying...

**Comment by jarimatti-q** [comment]
There is a config and it's enabled, but so far it didn't show any build jobs there. I'll see if that has changed now.

**Comment by jarimatti-q** [comment]
I've marked this PR job as being required in the settings.

**Pull request merged**